### PR TITLE
task 1 create entry

### PR DESCRIPTION
**Pull request by chrisahn1**
code journal create entry task 1

### CODE REVIEW

**Comment by chrisahn1** [comment]
code journal local storage
code journal desktop and phone display
code journal input

**Comment by chrisahn1** [comment]
All tasks under task 6 is counted as task 6 so task 7 starts with verifying new entries that are saved in the local storage.

File: index.html [hunk 1]
```diff
@@ -13,11 +13,56 @@
   <link rel="stylesheet" href="css/reset.css">
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
+  <style>
+    @media screen and (min-width: 568px) {
```
Comment (BrettAlbright) [inline index.html]: make sure to move the media queries into the styles.css file

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
looks good! the only updates are to move the media queries and to change push to unshift for moving the entries into the array

**Review by BrettAlbright — APPROVED** [review]
👍🏼

**Pull request merged**